setup/vault-server/entrypoint.sh

```diff
@@ -68,22 +70,22 @@ vault token create \
 ########## STATIC SECRETS ###########
 #####################################
 
-# enable a kv-v2 secrets engine, passing in the path parameter
+# Enable a kv-v2 secrets engine, passing in the path parameter
 # ref: https://www.vaultproject.io/docs/secrets/kv/kv-v2
 vault secrets enable -path=kv-v2 kv-v2
 
-# seed the kv-v2 store with an entry our web app will use
+# Seed the kv-v2 store with an entry our web app will use
 vault kv put "${API_KEY_PATH}" "${API_KEY_FIELD}=my-secret-key"
 
 #####################################
 ########## DYNAMIC SECRETS ##########
 #####################################
 
-# enable a database secrets engine
+# Enable a database secrets engine
```

Comment:
Just noticed this should probably be "the" database secrets engine. Right? Since that's the official name of that engine?

Reply:
agreed 👍

Reply:
Also changed "Enable the kv-v2 ..."